src/components/tables/Captures/index.tsx

```diff
@@ -1,7 +1,84 @@
 import { Box } from '@mui/material';
-import EntityTable from 'components/tables/EntityTable';
+import Rows from 'components/tables/Captures/Rows';
+import EntityTable, {
+    getPagination,
+    SortDirection,
+} from 'components/tables/EntityTable';
+import { useQuery } from 'hooks/supabase-swr';
+import { useState } from 'react';
+import { TABLES } from 'services/supabase';
+
+interface LiveSpecQuery {
+    spec_type: string;
+    catalog_name: string;
+    updated_at: string;
+    connector_image_name: string;
+    id: string;
+}
+
+const tableColumns = [
+    {
+        field: 'catalog_name',
+        headerIntlKey: 'entityTable.data.entity',
+    },
+    {
+        field: 'connector_image_name',
+        headerIntlKey: 'entityTable.data.connectorType',
+    },
+    {
+        field: 'updated_at',
+        headerIntlKey: 'entityTable.data.lastUpdated',
+    },
+    {
+        field: null,
+        headerIntlKey: 'entityTable.data.actions',
+    },
+];
+
+const queryColumns = [
+    'spec_type',
+    'catalog_name',
+    'updated_at',
+    'connector_image_name',
+    'id',
+];
 
 function CapturesTable() {
+    const rowsPerPage = 10;
+    const [pagination, setPagination] = useState<{ from: number; to: number }>(
+        getPagination(0, rowsPerPage)
+    );
+    const [searchQuery, setSearchQuery] = useState<string | null>(null);
+    const [sortDirection, setSortDirection] = useState<SortDirection>('desc');
+    const [columnToSort, setColumnToSort] = useState<any>('updated_at');
```

Comment:
This is the stuff that is currently duplicated. Not the best approach but not the worst.